docs/source/how-to/use-rules-composites-and-canonizers.rst

```diff
@@ -410,17 +417,59 @@ mapping from module name to rule:
     print(list(model.named_modules()))
 
     # manually write a rule mapping:
-    composite = NameMapComposite([
+    name_map = [
         (['conv0'], ZBox(low, high)),
         (['conv1'], ZPlus()),
         (['linear0', 'linear1'], Epsilon()),
-    ])
+    ]
+    composite_name_map = NameMapComposite(name_map=name_map)
 
 Modules built using :py:class:`torch.nn.Sequential` without explicit names will have a
 number string as their name. Explicitly assigning a module to a parent module as
 an attribute will assign the attribute as the child module's name. Nested
 modules will have their names split by a dot ``.``.
 
+
+Composites can be further mixed either by using a
+:py:class:`~zennit.composites.NameLayerMapComposite` or a
+:py:class:`~zennit.composites.MixedComposite`:
+
+
+.. code-block:: python
+
+    from zennit.composites import NameLayerMapComposite
+
+    # matching order is same as list order
+    composite_name_layer_map = NameLayerMapComposite(
+        name_map=name_map, layer_map=layer_map,
+    )
+
+This creates a composite which will in turn create a
+:py:class:`~zennit.composites.NameMapComposite` and a
+:py:class:`~zennit.composites.LayerMapComposite` instance.
+The created composite will first try to match for a specific layer
+name by applying the mapping from the instantiated
+:py:class:`~zennit.composites.NameMapComposite`.
+If None are found, the matching process continues with the
+:py:class:`~zennit.composites.LayerMapComposite`.
+
+
+.. code-block:: python
+
+    from zennit.composites import MixedComposite
+
+    # matching order is same as list order
+    composites = [composite_name_map, composite_special_first_layer]
+    # create an instance from list of composites
+    composite_mixed = MixedComposite(composites=composites)
+
+This creates a composite which will first try to match for a specific layer
+name by applying the respective mappings of each composite in the given list.
+In the example above, if a layer name match is successful, it registers the
+hook from `composite_name_map`. If no matching name found, the matching process
+continues with `composite_special_first_layer`.
```

Comment:
In ReST, inline literals (i.e. inline code) uses [double ticks](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#inline-literals).

Reply:
fixed